src/client/services/conversation/share-activity/share-activity-base.js

```diff
@@ -404,12 +423,29 @@ var ShareActivityBase = SparkBase.extend(
   },
 
   /**
+   * @param {Array} items
+   * @param {string} mimeType
+   * @private
+   * @returns {boolean}
+   */
+  _itemContainsActionWithMimeType: function _itemContainsActionWithMimeType(items, mimeType) {
+    return some(map(items, function doSomeActionsContainMimeType(item) {
```

Comment:
There are two, but it looks a bit different without the arrow functions.  The arrow functions broke the web-client. :smile: